crates/types/src/nodes_config.rs

```diff
@@ -59,8 +58,8 @@ pub struct NodesConfiguration {
     cluster_name: String,
     // flexbuffers only supports string-keyed maps :-( --> so we store it as vector of kv pairs
     #[serde_as(as = "serde_with::Seq<(_, _)>")]
-    nodes: HashMap<PlainNodeId, MaybeNode>,
-    name_lookup: HashMap<String, PlainNodeId>,
+    nodes: HashMap<PlainNodeId, MaybeNode, Xxh3Builder>,
+    name_lookup: HashMap<String, PlainNodeId, Xxh3Builder>,
```

Comment:
Unrelated: Should we define a type alias for this `HashMap` type?

Reply:
I like the clarity since it should not be used everywhere. Perhaps at some point I'll define alias types for specific use-case patterns (e.g. mapping node-id -> T, etc.)